desktop/appcontainer/ToastSource.js

```diff
@@ -53,13 +56,18 @@ export class ToastSource {
      * If non-default values are needed for a toaster, a different method must be used.
      *
      * @param {string} [position] - see Blueprint Position enum for allowed values.
+     * @params {Element} [containerRef] - see Blueprint 'container' parameter
      */
-    getToaster(position = Position.BOTTOM_RIGHT) {
-        const toasters = this._toasters;
+    getToaster(position, containerRef) {
+        const toasters = this._toasters,
+            container = containerRef ? containerRef.getDOMNode() : document.body,
+            containerId = containerRef ? containerRef.xhId : 'viewport',
```

Comment:
The need for a unique id to recognize the toaster definitely argues for asking the user pass in a Component.  Got it! 🌮

Reply:
Yes, but your point about the comment on line 59 is well taken. E.g. 
`[containerRef] component which passes its underlying DOM node as Blueprint's containerId params, and is used to set a unique ID for this Toaster instance